codec/any_test.go

```diff
@@ -53,7 +52,7 @@ func TestMarshalAny(t *testing.T) {
 	require.Equal(t, kitty, animal)
 
 	// nil should fail
-	registry = NewTestInterfaceRegistry()
+	_ = NewTestInterfaceRegistry()
 	err = cdc.UnmarshalInterface(bz, nil)
```

Comment:
This `_ = NewTestInterfaceRegistry()` doesn't seem necessary, and could be removed.
```suggestion
	err = cdc.UnmarshalInterface(bz, nil)
```